marklogic-data-hub-central/ui/e2e/cypress.json

```diff
@@ -3,12 +3,17 @@
     "video": true,
     "viewportWidth": 1680,
     "viewportHeight": 1050,
+    "numTestsKeptInMemory": 1,
```

Comment:
I don't think this change is needed. e2e test failures have not been due to high memory use.